extension/chrome/settings/inbox/inbox-modules/inbox-active-thread-module.ts

```diff
@@ -5,24 +5,49 @@
 import { Bm, BrowserMsg } from '../../../../js/common/browser/browser-msg.js';
 import { FactoryReplyParams, XssSafeFactory } from '../../../../js/common/xss-safe-factory.js';
 import { GmailParser, GmailRes } from '../../../../js/common/api/email-provider/gmail/gmail-parser.js';
-import { Str, Url, UrlParams } from '../../../../js/common/core/common.js';
+import { Url, UrlParams } from '../../../../js/common/core/common.js';
 
 import { ApiErr } from '../../../../js/common/api/shared/api-error.js';
 import { BrowserMsgCommonHandlers } from '../../../../js/common/browser/browser-msg-common-handlers.js';
 import { Buf } from '../../../../js/common/core/buf.js';
 import { Catch } from '../../../../js/common/platform/catch.js';
 import { InboxView } from '../inbox.js';
 import { Lang } from '../../../../js/common/lang.js';
-import { Mime } from '../../../../js/common/core/mime.js';
 import { Ui } from '../../../../js/common/browser/ui.js';
 import { ViewModule } from '../../../../js/common/view-module.js';
 import { Xss } from '../../../../js/common/platform/xss.js';
 import { Browser } from '../../../../js/common/browser/browser.js';
 import { Attachment } from '../../../../js/common/core/attachment.js';
+import { LoaderContextInterface } from '../../../../js/common/loader-context-interface.js';
+
+class LoaderContext implements LoaderContextInterface {
+  public constructor(private readonly factory: XssSafeFactory, public renderedMessageXssSafe: string | undefined, public renderedAttachments: string[]) {}
+
+  public renderPlainAttachment = (a: Attachment) => {
+    // todo: render error argument
```

Comment:
We can do this in a separate issue, as the `settings/inbox` page isn't a priority now.